x/auth/ante/sigverify.go

```diff
@@ -305,11 +338,15 @@ func DefaultSigVerificationGasConsumer(
 		meter.ConsumeGas(params.SigVerifyCostSecp256k1, "ante verify: secp256k1")
 		return nil
 
-	case multisig.PubKeyMultisigThreshold:
-		var multisignature multisig.AminoMultisignature
-		legacy.Cdc.MustUnmarshalBinaryBare(sig, &multisignature)
-
-		ConsumeMultisignatureVerificationGas(meter, multisignature, pubkey, params)
+	case multisig.PubKey:
+		multisignature, ok := sig.Data.(*signing.MultiSignatureData)
+		if !ok {
+			return fmt.Errorf("expected %T, got, %T", &signing.MultiSignatureData{}, sig.Data)
```

Comment:
this case is not covered

Reply:
This is already covered here, would you want me to add more tests https://github.com/cosmos/cosmos-sdk/blob/30b23cc5be20540d9524b9bdab637f630f336fb0/x/auth/ante/sigverify_test.go#L95